### PR TITLE
A test case for presence corruption with the eventual send candidate.

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
This fails with the new eventual send library, and succeeds in the current state.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -28,6 +28,13 @@
         "@agoric/transform-bang": "^0.3.1",
         "@babel/generator": "^7.5.5",
         "esm": "^3.2.5"
+      },
```
Comment (katelynsills) [inline package-lock.json]: what do the package-lock.json changes do?

Comment (Chris-Hibbert) [reply package-lock.json]: I thought I had reverted this. Let me try again.
The bug this test case is meant to reveal is provoked by the current draft of the eventual-send code. 

The test case is intended to succeed in the current master, but be ready to fail if the bug in the draft eventual-send code isn't fixed.

File: test/presenceCorruption/test-demos.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { test } from 'tape-promise/tape';
```
Comment (katelynsills) [inline test/presenceCorruption/test-demos.js]: Can we rename this from test-demos.js?

Comment (Chris-Hibbert) [reply test/presenceCorruption/test-demos.js]: Oh yeah. done.

**Comment by Chris-Hibbert** [comment]
approved by all; merging
